doc/cephfs/mount-prerequisites.rst

```diff
@@ -6,6 +6,8 @@ You can use CephFS by mounting it to your local filesystem or by using
 FUSE`_. Both have their own advantages. Read the following section to
 understand more about both of these ways to mount CephFS.
 
+Mounting CephFS requires superuser privileges.
+
```

Comment:
The change feels out of place. I think following would be better.

```suggestion
FUSE`_. Both have their own advantages and both require superuser
privileges to mount. Read the following section to understand more about
both of these ways to mount CephFS.
```

Also, replacing [`fuse.conf option`section](https://docs.ceph.com/en/pacific/cephfs/mount-using-fuse/#fuse-conf-option) by a note point (for informing and convenience of `ceph-fuse` users) would be great.

Reply:
Done.